Embrace Unity iOS Interface/EmbraceManager.swift

```diff
@@ -0,0 +1,494 @@
+    }
+    
+    static func addUserPersona(persona: String) -> Bool {
+        do {
+            try Embrace.client?.metadata.add(persona: persona, lifespan: .session)
+            return true
+        } catch let error {
+            return false
+        }
+    }
+    
+    static func clearUserPersona(persona: String) -> Bool {
+        do {
+            try Embrace.client?.metadata.remove(persona: PersonaTag(persona), lifespan: .session)
+            return true
+        } catch let error {
+            return false
+        }
+    }
+    
+    static func clearAllUserPersonas() -> Bool {
+        do {
+            try Embrace.client?.metadata.removeAllPersonas()
+            return true
+        } catch let error {
+            return false
+        }
+    }
+    
+    static func addResource(key: String, value: String, lifespan: MetadataLifespan) -> Bool {
+        do {
+            try Embrace.client?.metadata.addResource(
+                key: key, value: value, lifespan: lifespan)
+            return true
+        } catch let error {
+            
+        }
+        
+        return false
+    }
+    
+    static func addSessionProperty(key: String, value: String, permanent: Bool) -> Bool {
+        do {
+            let lifespan: MetadataLifespan = permanent ? .permanent : .session
+            try Embrace.client?.metadata.addProperty(key: key, value: value, lifespan: lifespan)
+            return true
+        } catch let error {
+            return false
+        }
+    }
+    
+    static func removeSessionProperty(key: String) -> Bool {
+        do {
+            try Embrace.client?.metadata.removeProperty(key: key, lifespan: .permanent)
+            try Embrace.client?.metadata.removeProperty(key: key, lifespan: .session)
+            return true
+        } catch let error {
+            return false
+        }
+    }
+    
+    static func logMessageWithSeverityAndProperties(
+        message: String,
+        severity: String,
+        properties: [String: String]) {
+            Embrace.client?.log(message,
+                severity: convertStringToLogSeverity(from: severity),
+                attributes: properties
+            )
+    }
+    
+    static func setUserAsPayer() -> Bool {
+        do {
+            try Embrace.client?.metadata.add(persona: "payer")
+            return true
+        } catch let error {
+            return false
+        }
+    }
+    
+    static func clearUserAsPayer() -> Bool {
+        do {
+            try Embrace.client?.metadata.remove(persona: "payer", lifespan: .session)
+            return true
+        } catch let error {
+            return false
+        }
+    }
+    
+    static func startView(viewName: String) -> String? {
+        let span = Embrace.client?.buildSpan(name: "emb-screen-view")
+            .setAttribute(key: "view.name", value: viewName)
+            .setAttribute(key: "emb.type", value: "ux.view")
+            .startSpan()
```

Comment:
Not a big deal, but we exposed a new target that is `EmbraceSemantics` that has some of this values.
For this particular case, all those values are provided:
```swift
public extension SpanType {
    static let view = SpanType(ux: "view")
}

public extension SpanSemantics {
    struct View {
        public static let name = "emb-screen-view"
        public static let keyViewTitle = "view.title"
        public static let keyViewName = "view.name"
    }
}
```
So, I think this would look like something like this:
```swift
let span = Embrace.client?.buildSpan(name: SpanSemantics.View.name)
    .setAttribute(key: SpanSemantics.View.keyViewName, value: viewName)
    .setAttribute(key: SpanSemantics.keyEmbraceType, value: "ux.view")
    .startSpan()
```